tests/core/tests/admin_integration/test_admin_integration.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+from core.admin import BookAdmin
+from django.contrib.auth.models import User
+
+from import_export.formats.base_formats import DEFAULT_FORMATS
+
+
+class AdminTestMixin(object):
```

Comment:
This file contains a mixin and no tests, therefore I think the file should be renamed as `mixins.py`